content/en/deployment_gates/setup.md

```diff
@@ -0,0 +1,372 @@
+                              exit 1
+                          fi
+                          
+                          # At this point, we have received a 200 status code. So, we check the gate status returned
+                          GATE_STATUS=$(echo "$RESPONSE_BODY" | jq -r '.data.attributes.gate_status')
+                          
+                          if [[ "$GATE_STATUS" == "pass" ]]; then
+                              echo "Gate evaluation PASSED"
+                              exit 0
+                          else
+                              echo "Gate evaluation FAILED"
+                              exit 1
+                          fi
+                        done
+
+                        # If we arrive here, it means that we received several 5xx errors from the API. To not block deployments, we can treat this case as a success
+                        echo "All retries exhausted, but treating 5xx errors as success."
+                        exit 0
+```
+
+* The analysis template can receive arguments from the Rollout resource. In this case, the arguments are `service`, `env` and any other optionnal fields needed. For more information, see the [official Argo Rollouts docs][3].
+* The `ttlSecondsAfterFinished` field removes the finished jobs after 5 minutes.
+* The `backoffLimit` field is set to 0 as the job might fail in case that the gate evaluation fails, and it should not be retried in that case.
+
+Once you have created the analysis template, reference it from the Argo Rollout strategy:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+  labels:
+    tags.datadoghq.com/service: transaction-backend
+    tags.datadoghq.com/env: dev
+spec:
+  replicas: 5
+  strategy:
+    canary:
+      steps:
+        ...
+        - analysis:
+            templates:
+              - templateName: datadog-job-analysis
+                clusterScope: true # only needed for cluster analysis
+            args:
+              - name: env
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.labels['tags.datadoghq.com/env']
+              - name: service
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.labels['tags.datadoghq.com/service']
+              - name: version # only required if one or more APM Faulty Deployment Detection rules are evaluated.  
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.labels['tags.datadoghq.com/version']
+        - ...
+```
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: https://argo-rollouts.readthedocs.io/en/stable/features/analysis/#analysis-progressive-delivery
+[3]: https://argo-rollouts.readthedocs.io/en/stable/features/analysis/#analysis-progressive-delivery
+[4]: https://argo-rollouts.readthedocs.io/en/stable/features/analysis/#analysis-template-arguments
```

Comment:
I'd recheck all of these, there's a duplicate and the first doesn't seem to be used